include/library/spdm_common_lib.h

```diff
@@ -278,6 +278,15 @@ libspdm_return_t libspdm_init_context(void *spdm_context);
  */
 void libspdm_reset_context(void *spdm_context);
 
+/**
+ * Free the SPDM context: if there are some memory need free, the function will free them.
```

Comment:
> Free the memory of contexts within the SPDM context. These are typically contexts whose memory has been allocated by the cryptography library. This function does not free the SPDM context itself.

Reply:
Thanks. I have changed the function description.